styles/variables.scss

```diff
@@ -12,4 +12,9 @@ $dark-brown: rgb(46, 30, 20);
 // A variable for RGB 136, 157, 53
 $light-green: rgb(136, 157, 53);
 // A variable for RGB 53, 101, 137
-$dark-blue: rgb(53, 101, 137);
\ No newline at end of file
+$dark-blue: rgb(53, 101, 137);
+$light-brown: rgba(138 114 39);
+$white:rgba(243 241 236);
+$light-blue: rgba(198 225 238);
```

Comment:
`$light-blue`, `light-greed` and `dark-blue` repeated 😅

Reply:
changed it to whitehover. So its not repeated anymore

Reply:
not sure why it didn't show that change